### ISSUE TITLE
Ratatui.rs widget showcase

### ISSUE THREAD

**Issue by joshka** [gin66/tui-logger#52]
Hey there,
I'm putting together a widget showcase page at https://ratatui.rs/showcase/third-party-widgets/ and would like to add tui-logger. The demo right now seems a bit complex to use as it shows all the possible features. Would you be interested in creating a simpler demo that just shows the main aspect in a small area like the examples we have on the site already? I'm happy to create the actual gif if you have a good simple demo (though the process is fairly simple - see https://github.com/ratatui-org/website/tree/main/code/widget-showcase-third-party)

**Comment by gin66** [comment]
Thanks for the invitation to add a demo to ratatui’s awesome web presence.
I will think about a demo, but can work on it earliest after Xmas.

In addition I am nterested to see your process to create the gif. My own process has been quite tedious and inconvenient.

**Comment by joshka** [comment]
Nothing magic to it https://github.com/ratatui-org/website/tree/main/code/widget-showcase-third-party
TL;DR https://github.com/charmbracelet/vhs + small Ratatui app

**Comment by gin66** [comment]
seems @joshka haa done it already. Big thanks

**Comment by joshka** [comment]
![tui-logger](https://github.com/gin66/tui-logger/assets/381361/a2e6925c-21c5-4320-9fca-264931a01d6b)
From https://github.com/gin66/tui-logger/pull/57
Incorporated into the site at https://ratatui.rs/showcase/third-party-widgets/#tui-logger--
